CONTRIBUTING.rst

```diff
@@ -21,7 +21,7 @@ Releasing
 
 A reminder for the maintainers on how to make a release.
 Make sure all your changes are committed.
```

Comment:
Should this specify that the changes need to be merged into master, or is that obvious?

Reply:
Seems obvious to me. If you are updating/creating the release on master, and a change isn't yet merged, it won't be part of that release.